modules/investigating-kernel-crashes.adoc

```diff
@@ -74,7 +74,8 @@ endif::[]
 . Ensure that `kdump` has loaded a crash kernel by checking that the `kdump.service` has started and exited successfully and that `cat /sys/kernel/kexec_crash_loaded` prints `1`.
 
 == Enabling kdump on day-1
-The `kdump` service is intended to be enabled per-node to debug kernel problems. It is not recommended to enable `kdump` on all of your nodes in the cluster. Although machine-specific machine configs are not yet supported, you can perform the previous steps through a `systemd` unit in a `MachineConfig` object on day-1 and have kdump enabled on all nodes in the cluster. You can create a machine config object and inject that object into the set of manifest files used by Ignition during cluster setup. See "Customizing nodes" in the _Installing -> Installation configuration_ section for more information and examples on how to use Ignition configs.
+The `kdump` service is intended to be enabled _per-node_ to debug kernel problems. Because there are costs to having `kdump` enabled, and these costs accumulate with each additional `kdump`-enabled node, it is recommended that `kdump` only be enabled on each node _as needed_. Potential costs of enabling `kdump` on each node include less available RAM due to memory being reserved for the crash kernel, node unavailability while the kernel is dumping the core, additional storage space being used to store the crash dumps, not being supported with Red Hat production SLAs due to the `kdump` service being in link:https://access.redhat.com/support/offerings/techpreview[Technology Preview], etc.
+If you are aware of the downsides and trade-offs of having the `kdump` service enabled, it is possible to enable `kdump` in a cluster-wide fashion. Although machine-specific machine configs are not yet supported, you can perform the previous steps through a `systemd` unit in a `MachineConfig` object on day-1 and have kdump enabled on all nodes in the cluster. You can create a machine config object and inject that object into the set of manifest files used by Ignition during cluster setup. See "Customizing nodes" in the _Installing -> Installation configuration_ section for more information and examples on how to use Ignition configs.
```

Comment:
Thanks so much for adding these clarifications, @kelvinfan001. A few comments/suggestions:
- We use underscores very sparingly for emphasis (see [table](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference) for reference). Hence, I would remove them with `_per-node_ and _as needed_ here. Also, per-node doesn't need a hyphen.
- For readability, I would break this out into two paragraphs, use a bulleted list for costs, and drop "etc.".

Paragraph #1: The `kdump` service....These potential costs include:

- Less available RAM due to memory being reserved for the crash kernel.
- Node unavailability while the kernel is dumping the core.
- Additional storage space being used to store the crash dumps.
- Not being production-ready because the `kdump` service is in link:https://access.redhat.com/support/offerings/techpreview[Technology Preview].

Paragraph #2: If you are aware...

Reply:
This would make things easier to read, thanks!

Reply:
This looks great, @kelvinfan001 - thanks for the updates. Glad we removed the TP/SLA wording here because I just realized it is already described in the previous section.

LGTM!